.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - name: Install dependencies
+        run: npm i
+      - name: Type Check
+        run: npm run tsc
```

Comment:
Actually, IMHO it should be handle as another pull request because there is no relation to its feature, *Add Mined Block list in account page*.

Reply:
It was difficult to type check, so I #102 closed and merged to this branch.

Reply:
Yes, I am strongly sympathetic and agree with your opinion, difficulty to check types. But what I wanted to say was, what relationships exist between creating GitHub Actions Workflow and adding *Mined Block list in account page*.

Reply:
Ok I 'll remove this file.